orsopy/fileio/data_source.py

```diff
@@ -4,31 +4,57 @@
 
 # author: Andrew R. McCluskey (arm61)
 
-import enum
+# import enum
 from typing import Optional, Dict, List, Union, Literal
 from dataclasses import field, dataclass
-import datetime
 
 from .base import File, Header, ValueRange, Value, ValueVector, Person
 
 
 @dataclass(repr=False)
 class Experiment(Header):
-    """A definition of the experiment performed."""
+    """
+    A definition of the experiment performed.
+
+    :param title: Proposal or project title.
+    :param instrument: Reflectometer identifier.
+    :param start_date: Start date for the experiment.
+    :param probe: Radiation probe, either :code:`'neutrons'` or
+        :code:`'x-rays'`.
+    :param facility: Facility where the experiment was performed.
+    :param proposalID: Identifier for experiment at a facility.
+    :param doi: Digital object identifier for the experiment, possibly
+        provided by the facility.
+    """
     title: str
     instrument: str
-    date: datetime.datetime
-    probe: Union[Literal["neutrons", "x-rays"]]
+    start_date: str
```

Comment:
Is there a reason why datetime.datetime was removed? Fine to have string, but makes it non-validateable.